lightning/src/ln/functional_tests.rs

```diff
@@ -3724,12 +3724,17 @@ fn test_peer_disconnected_before_funding_broadcasted() {
 		assert_eq!(nodes[0].tx_broadcaster.txn_broadcasted.lock().unwrap().len(), 0);
 	}
 
-	// Ensure that the channel is closed with `ClosureReason::DisconnectedPeer` when the peers are
-	// disconnected before the funding transaction was broadcasted.
+	// Ensure that the channel is closed if the peer does not reconnect within the timeout period
+	// when the peers are disconnected before the funding transaction was broadcasted.
```

Comment:
This is a bit of a mouthful now. Instead of modifying the comment, let's move it above `check_closed_event`.